From f67db5e2db64bfc92f1e4ccb31660a11c9602a28 Mon Sep 17 00:00:00 2001
From: "Tan, Tee Min" <tee.min.tan@intel.com>
Date: Sun, 1 Dec 2019 13:42:08 +0800
Subject: [PATCH 1/3] REVERTME: net: stmmac: Workaround change PTP clock rate
 to 204.86Mhz

PCH GBE clocksource is not exact 200Mhz. For now, changed clock rate
to 204.86Mhz in order to have the accurate 1-sec PPS output.

Signed-off-by: Tan, Tee Min <tee.min.tan@intel.com>
---
 drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c b/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c
index 5928a1b3be15..09522fe0ea3d 100644
--- a/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c
+++ b/drivers/net/ethernet/stmicro/stmmac/stmmac_pci.c
@@ -341,7 +341,7 @@ static int ehl_sgmii_data(struct pci_dev *pdev,
 	plat->phy_interface = PHY_INTERFACE_MODE_SGMII;
 
 	/* Set PTP clock rate for EHL as 200MHz */
-	plat->clk_ptp_rate = 200000000;
+	plat->clk_ptp_rate = 204860000;
 
 	return ehl_common_data(pdev, plat);
 }
-- 
2.17.0

